packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -240,6 +262,22 @@ export default defineComponent<KottiButton.PropsInternal>({
 				border-bottom-color: var(--button-main-color-dark);
 				border-left-color: var(--button-main-color-dark);
 			}
+			&.kt-button--is-toggle-on {
+				color: var(--text-04) !important;
```

Comment:
can you actually try without important and see if it works

i think you have higher specifity here, there's no way this doesn't apply and take precedence